### ISSUE TITLE
Active State for menus in 1.13.0 with Smoothness Theme

### ISSUE THREAD

**Issue by noyearzero** [jquery/jquery-ui#2033]
The active state for menu items in the default theme has a blue background and all the other themes I checked makes the active item stand out visuall... but the smoothness theme only adds a light gray border.  It's also different from how it looked in 1.12.  I'm not sure if this is a bug or intentional.  Either way, I think it could be improved.

**Comment by noyearzero** [comment]
![jquery ui default](https://user-images.githubusercontent.com/2496224/145845854-78b4ae5d-4309-466c-8cbb-aada8cf45630.png)
![jquery ui smoothness](https://user-images.githubusercontent.com/2496224/145845895-1a8e44c1-015e-4698-9c4e-f804b5b5a9fc.png)

**Comment by noyearzero** [comment]
@mgol I'm wondering why the label "Behavior shared with 1.12" was added.  For me it's different in that version as shown in the screenshots.

**Comment by mgol** [comment]
Hey! Is the first screenshot supposed to be jQuery UI 1.12.1? I have that version in the Smoothness theme downloaded locally and it looks just like in 1.13.0, i.e. your second screenshot. Your first screenshot looks like the Base theme.

**Comment by noyearzero** [comment]
Oh yeah.  I was having a hard time finding an example of Smoothness.  The thing I was working on would have been quite difficult to revert, so I used the base.  But I was able to find an example of what 1.12.1 looked like

**Comment by noyearzero** [comment]
Yeah, you might be right.  I might have been a modified version of Smoothness because this issue previous existed.
